libs/core/src/components/sage-link/stories/sage-link.stories.mdx

```diff
@@ -1,12 +1,14 @@
 import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
 import { html, render } from 'lit-html';
+import { spread } from '@open-wc/lit-helpers';
 
 import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
 
 <Meta title="webcomponents/sage-link" component="sage-link" argTypes={extractArgTypes('sage-link')} args={{ href: '#some-anchor' }} />
 
-export const Template = (args) => html`<sage-link external=${args.external} font-size=${args.fontSize} href=${args.href} variant=${args.variant}></sage-link>`;
-export const TemplateWithSlot = (args) => html` <sage-link external=${args.external} font-size=${args.fontSize} href=${args.href} variant=${args.variant}>${args.slot}</sage-link>`;
+export const Template = (args) => html`<sage-link ${spread(args)}></sage-link>`;
```

Comment:
I'm not opposed to using this if it's only for Storybook and provides us benefits in our docs. I like the approach to being more specific, personally. The question I have for use of `spread` is that do we _need_ to add it? Could we come up with an alternate solution instead, e.g. having one/the initial story dedicated to sandboxing and all other stories dedicated to common patterns? Furthermore, should common patterns be a section in our docs page?

Reply:
I'm torn too. I don't mind using `spread` but I do also wonder if we could just split up our code examples and take on the addition of a template for each example. Maintaining too many templates could be a good "canary in the coal mine" for when a component is getting too complex. 

That said, I don't mind if we want to use `spread` instead of adding multiple templates.

Reply:
I like the idea, but I can see where confusion could arise. Currently we have the issue where the attributes are showing in the DOM due to our main template containing all props. Because of that, it's helpful for that template but I also like the explicit way of setting them to reduce confusion as well. I don't mind multiple template files since it could ensure correct usage.

Reply:
Based on the conversation here, it seems as if there's no real definitive direction that we would like to pursue regarding adding this helper. I'm going to vote that we dismiss using `spread` and come up with an alternate solution for now like creating different stories for different use cases. If that gets too out of hand, then maybe we can reconsider `spread` usage in the future. Any objections?

I am more than happy to spend some time looking into an avenue for how this can look in the docs as long as it doesn't interfere with Quinton's work on docs improvement.

Reply:
Sounds good to me, love having this PR here (even if its closed) so we can come back to spread if we ever change our minds. Thank you for digging into this!